src/appimagetool.c

```diff
@@ -522,10 +522,34 @@ main (int argc, char *argv[])
     travis_tag = getenv("TRAVIS_TAG");
     char* travis_pull_request;
     travis_pull_request = getenv("TRAVIS_PULL_REQUEST");
+
+    /* Parse GitHub Actions environment variables.
+     * https://docs.github.com/en/actions/learn-github-actions/variables
+     * GITHUB_REPOSITORY: The owner and repository name. For example, octocat/Hello-World.
+     * GITHUB_REPOSITORY_OWNER: The repository owner's name. For example, octocat.
+     */
+
     /* https://github.com/probonopd/uploadtool */
     char* github_token;
     github_token = getenv("GITHUB_TOKEN");
 
+    // Construct the repository name from github_repository and github_repository_owner
+    // by removing the github_repository_owner from the beginning of github_repository
+    // and the slash that follows it
+    char* github_repository = getenv("GITHUB_REPOSITORY");
```

Comment:
You don't really need to create these variables in the function's root scope. You could set up the variables and the parsing below starting in line 921.

I know the Travis parsing is located here, too, but I guess I'll soon send a PR to remove it.

Reply:
As above; I did this to fit in with the existing code.

Reply:
If you feel strongly above this, then feel free to edit the code in this PR; it works for me as-is and I don't want to break it.